src/triage/component/catwalk/model_testers.py

```diff
@@ -39,6 +39,11 @@ def __init__(
             training_metric_groups=evaluator_config.get("training_metric_groups", []),
         )
 
+        try:
+            self.subsets = evaluator_config["subsets"]
+        except:
```

Comment:
again, `KeyError`?

I would think this could be even plainer:

```
self.subsets = evaluator_config.get('subsets')
```

Reply:
totally embarrassing, but I didn't know about `.get()`. That's much better.